Automattic/studio
# feat: Assistant renders Markdown tables

Fixes https://github.com/Automattic/dotcom-forge/issues/7720.

## Proposed Changes

- Leverage `remark-gfm` to parse and render Markdown table syntax to improve
  legibility of assistant responses.
- Allow horizontal scrolling of assistant messages to ensure wide content, like
  large tabular data, is visible.

## Testing Instructions



1. Request tabular data containing a large number of columns from the AI assistant.
1. Verify a styled table is rendered and is scrollable to view all information.

<img width="1012" alt="markdown-table" src="https://github.com/Automattic/studio/assets/438664/252256b4-f954-467a-86b7-6f8a05c5e4f3">

## Pre-merge Checklist
